platform-coop-toolkit/coop-library
# fix: enable custom filter order (resolves #168)

## Description

With https://github.com/platform-coop-toolkit/coop-library-framework/pull/295 in place, terms should display sorted by numeric order, then name. This has the impact of allowing terms to be sorted numerically at the parent level (where an explicit numeric order has been set) while reverting to alphabetical order at the child level.

## Steps to test

1. Ensure that both this PR and https://github.com/platform-coop-toolkit/coop-library-framework/pull/295 are checked out.
2. Apply custom order to the top-level topics, as defined in #168.
3. Browse all resources.

**Expected behavior:** Topics are sorted by custom order at the parent level and alphabetically at the child level.

## Additional information

Not applicable.

## Related issues

- Depends on https://github.com/platform-coop-toolkit/coop-library-framework/pull/295.
- Resolves #168.
